lib/util/cache/repository/impl/base.ts

```diff
@@ -41,26 +56,30 @@ export abstract class RepoCacheBase implements RepoCache {
     this.oldHash = oldCache.hash;
   }
 
-  private restoreFromRev11(oldCache: RepoCacheRecordV11): void {
-    this.data = oldCache.data;
-  }
-
-  private restoreFromRev10(oldCache: RepoCacheRecordV10): void {
-    delete oldCache.repository;
-    delete oldCache.revision;
-    this.data = oldCache;
+  private async restoreFromRev13(oldCache: RepoCacheRecordV13): Promise<void> {
+    if (oldCache.fingerprint !== this.fingerprint) {
+      logger.debug('Repository cache fingerprint is invalid');
+      return;
+    }
+    await this.restoreFromRev12(oldCache);
   }
 
   async load(): Promise<void> {
     try {
-      const data = await this.read();
-      if (!is.string(data)) {
+      const rawOldCache = await this.read();
```

Comment:
any reason for this rename?

Reply:
In order to avoid confusion with `this.data`